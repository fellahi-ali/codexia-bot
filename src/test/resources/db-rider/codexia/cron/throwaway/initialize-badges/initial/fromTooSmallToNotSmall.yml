codexia_project:
    -
        id: 8
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 123
        coordinates: "test-project1/test-repo1"
        author: "some-first-author2"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

codexia_review:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 8
        author: TOO_SMALL
        reason: 840
        text: "The repo is too small (LoC is 840)."
    -
        id: 2
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 8
        author: TOO_SMALL
        reason: 6000
        text: "The repo is not small anymore (LoC is 6000)."

codexia_badge:
    -
        codexia_project_id: 8
        badge: "bad"
        deleted_at: null

