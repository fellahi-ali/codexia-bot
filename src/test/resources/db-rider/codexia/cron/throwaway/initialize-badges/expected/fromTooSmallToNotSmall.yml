codexia_badge:
    -
        codexia_project_id: 8
        badge: "bad"
        deleted_at: regex:^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d{1,6}$
